programs/management/commands/import_urgent_need_config.py

```
@@ -0,0 +1,364 @@
+        ]:
+            setattr(need, field, config.get(field, default))
+
+        if "phone_number" in config:
+            need.phone_number = config.get("phone_number")
+
+        # FPL year
+        if fpl_data := config.get("fpl"):
+            year = fpl_data.get("year")
+            period = fpl_data.get("period", year)
+            if year is None:
+                raise CommandError("If 'fpl' is provided, it must include 'year'")
```

Ruff (0.15.0)

[warning] Avoid specifying long messages outside the exception class

(TRY003)